### DISCUSSION TITLE
General a resource bundle with 100 patients belonging to a particular organization

### DISCUSSION THREAD

**Discussion by vineetsaurabh** [synthetichealth/synthea#1463] [Q&A]
I want to generate 100 patients belonging to a particular organization. In the resource bundle, I want patients, organization, and maybe practitioners. Any help will be highly appreciated!

I found a similar question:
Restricting data to X number of Hospitals #1327

**Comment by jawalonoski** [comment]
Synthea doesn't have any built-in configuration settings to do exactly what you're asking.

In addition to considering the response in #1327, you basically will have to post-process the resulting FHIR data with a script (or whatever method you prefer) to repackage the FHIR data into bundles as you specified. It might help to use the FHIR bulk format (ndjson) to do that, since they are already spliced by resource type.

You can use the FHIR bulk format by specifying `--exporter.fhir.bulk_data=true` on the command line, or editing your `src/main/resources/synthea.properties` file.
